kuksa-client/kuksa_client/cli_backend/__init__.py

```diff
@@ -0,0 +1,46 @@
+########################################################################
+# Copyright (c) 2022 Robert Bosch GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+########################################################################
+
+import importlib.resources
+
+class Backend:
+    def __init__(self, config):
+        self.serverIP = config.get('ip', "127.0.0.1")
+        self.serverPort = config.get('port', 8090)
+        try:
+            self.insecure = config.getboolean('insecure', False)
+        except AttributeError:
+            self.insecure = config.get('insecure', False)
+        with importlib.resources.path('kuksa_certificates', '') as default_cert_path:
```

Comment:
At least for me this causes problems if using Python 3.9, but works well if using Python 3.10. Seems the implementation of importlib has changed a bit recently. Maybe not a problem if we do not work on Python 3.9, but do we need to mention it somewhere? (or has anyone else successfully run it on Python 3.9)

```
Default tokens directory: /home/erik/kuksa.val/kuksa_certificates/jwt

  File "/home/erik/kuksa.val/kuksa-client/kuksa_client/cli_backend/__init__.py", line 29, in __init__
    with importlib.resources.path('kuksa_certificates', '') as default_cert_path:
  File "/usr/lib/python3.9/contextlib.py", line 117, in __enter__
    return next(self.gen)
  File "/usr/lib/python3.9/importlib/resources.py", line 175, in _path_from_reader
    opener_reader = reader.open_resource(norm_resource)
  File "<frozen importlib._bootstrap_external>", line 995, in open_resource
IsADirectoryError: [Errno 21] Is a directory: '/home/erik/kuksa.val/kuksa-client/kuksa_certificates'

```

Reply:
Interesting. The only documentation I can find on `importlib.resources.path` 's `resource` parameter reads:

>  resource is the name of the resource to open within package; it may not contain path separators and it may not have sub-resources (i.e. it cannot be a directory).

It basically tells nothing on the expected behaviour when `resource == ''`.
I will look for a documented way of achieving what I want.